assets/js/payment-methods-demo/payment-methods/index.js

```diff
@@ -1,41 +1,25 @@
 /**
- * Internal dependencies
+ * External dependencies
  */
-import { paypalSvg } from './paypal';
-import { ccSvg } from './cc';
-
-const PaypalActivePaymentMethod = () => {
-	return (
-		<div>
-			<p>This is where paypal payment method stuff would be.</p>
-		</div>
-	);
-};
+import { __ } from '@wordpress/i18n';
 
-const CreditCardActivePaymentMethod = () => {
-	return (
-		<div>
-			<p>This is where cc payment method stuff would be.</p>
-		</div>
-	);
-};
+/**
+ * Internal dependencies
+ */
+import { PAYMENT_METHOD_NAME, StripeCreditCard } from './stripe';
+import { stripePromise } from '../stripe-utils';
 
-export const paypalPaymentMethod = {
-	id: 'paypal',
-	label: <img src={ paypalSvg } alt="" />,
-	stepContent: <div>Billing steps</div>,
-	activeContent: <PaypalActivePaymentMethod />,
-	edit: <PaypalActivePaymentMethod />,
-	canMakePayment: Promise.resolve( true ),
-	ariaLabel: 'paypal payment method',
-};
+const EditPlaceHolder = () => <div>TODO: Card edit preview soon...</div>;
 
-export const ccPaymentMethod = {
-	id: 'cc',
-	label: <img src={ ccSvg } alt="" />,
-	stepContent: null,
-	activeContent: <CreditCardActivePaymentMethod />,
-	edit: <CreditCardActivePaymentMethod />,
-	canMakePayment: Promise.resolve( true ),
-	ariaLabel: 'credit-card-payment-method',
+export const stripeCcPaymentMethod = {
+	id: PAYMENT_METHOD_NAME,
+	label: (
+		<strong>
+			{ __( 'Credit/Debit Card', 'woo-gutenberg-products-block' ) }
+		</strong>
+	),
+	activeContent: <StripeCreditCard />,
```

Comment:
I think content would be fine for naming - I think you used active because it's a tab, but I don't think thats important when it comes to naming things in the common API.

Reply:
I didn't just use `activeContent` because it's in a tab. I used it because it's the component to render when the _payment method_ is selected (and thus made active).  However, I do realize that we could probably just get away with `content` here since there is no other "content" type (other than label which is already distinctive). I'll change.